IntegrationTestProject/ProjectUnderTest/ProjectUnderTest/Foo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace ProjectUnderTest
+{
+    /* FxCop violations: 
+    *
+    * CA1801 (unused params)
+    * CA1822 (Bar can be static)
+    * CA2201 (Do not raise reserved exceptions)
+    * CA1303 (Do not pass literals as localized params)
+
+    * Also triggered: 
+    *   common-cs:InsufficientCommentDensity
+    *
+    * And
+    *   SonarLit S228 (do not use Console)
```

Comment:
Does the code contains rule violations for rules that are not enable in the quality profile? If so, it's not clear which ones.